spec/BBCoreSpec.js

```diff
@@ -867,8 +867,10 @@ describe("BBCore.videoRecorder", function() {
         setupMockApiRequest(result.withOptionsSuccess);
 
         bbCore.getVideoRecorder(opts, successCallbackSpy);
-
-        expect($.ajax.calls.argsFor(0)[0].data).toEqual(opts);
+        data = $.ajax.calls.argsFor(0)[0].data
+        for (var k in opts) {
+          expect(data[k]).toEqual(opts[k]);
```

Comment:
Iterating through keys because comparing objects via `toEqual` won't work as `getVideoRecorder` sticks `method` in the opts. Didn't see a built-in matcher to do this in Jasmine.